### PR TITLE
Angie config

### PR DESCRIPTION
**Pull request by angelica-viorato**

### CODE REVIEW

File: fn/develop/configs.md [hunk 1]
```diff
@@ -1,22 +1,115 @@
-# Function Config Vars
+# Use of environment variables
 
-There are three ways to get configuration variables into a function, all of which will be available as environment variables.
-These are ordered in order of preference, the later ones overriding the previous ones.
+If you want to use configuration variables into a function and make them available as environment variables, then you can set them using the Fn tool or the func.yaml file.
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: Suggested change: 

Do you want to use configuration variables into a function and make them available as environment variables? You can set configuration variables using the Fn CLI or the `func.yaml` file.

File: fn/develop/configs.md [hunk 2]
```diff
@@ -1,22 +1,115 @@
...
+ 
+By using the Fn tool, you can set configuration variables on the application or configuration level.
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: Suggested change:

By using the Fn CLI, you can set configuration variables on the application or function level.

File: fn/develop/configs.md [hunk 3]
```diff
@@ -1,22 +1,115 @@
...
 
-## (1) Application level configuration
+# Set environment variables
 
+The following three examples shows you how to set the configuration variables on different levels using the `connection-app` application. These are listed in order of preference, the later ones overrides the previous ones:
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: Suggested change: 

The following three examples show you how to set the configuration variables on different levels using the `connection-app` sample application. The configuration examples are listed in order of precedent, the later examples override the previous ones. For example, adding a function key/value pair overrides an application key/value pair with the same name.

File: fn/develop/configs.md [hunk 4]
```diff
@@ -1,22 +1,115 @@
...
+cmd: com.example.fn.Connection::getConnection
+config:
+  DB_DRIVER: mysqlDriver
+```
 See [Function file](func-file.md) for more info.
 
+(3) Function level configuration
 
-## (3) Function level configuration
+After you deploy your function, you can set configuration variables at the level configuration.
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: Suggested change:

..., you can set function level configuration variables. The variables only apply to the specific function identified.

File: fn/develop/configs.md [hunk 5]
```diff
@@ -1,22 +1,115 @@
...
+By using the Fn tool, you can set configuration variables on the application or configuration level.
 
-## (1) Application level configuration
+# Set environment variables
 
+The following three examples shows you how to set the configuration variables on different levels using the `connection-app` application. These are listed in order of preference, the later ones overrides the previous ones:
+
+
+(1) Application level configuration
+
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: Please add:

Configuration variables set at the application level apply to all functions contained in that application.

File: fn/develop/configs.md [hunk 6]
```diff
@@ -1,22 +1,115 @@
...
+    	return  password;        
+    }
+    
+    public String getConnection() {
+    	return "driver: " + getDriver() + "; url: " + url + "; user: " + getUser() + "; password: " + getPassword();
+    }
+}
 ```
+
+See [configuration-variables](../../fdks/fdk-java/examples/configuration-variables) for more info.
```
Comment (michael-w-williams) [inline fn/develop/configs.md]: See [configuration-variables example application](../../fdks/fdk-java/examples/configuration-variables) for more info.

File: fn/develop/configs.md (hunk 2 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Done

File: fn/develop/configs.md (hunk 1 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Done

File: fn/develop/configs.md (hunk 3 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Done

File: fn/develop/configs.md (hunk 4 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Done

File: fn/develop/configs.md (hunk 5 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Like

File: fn/develop/configs.md (hunk 6 above)
Comment (angelica-viorato) [reply fn/develop/configs.md]: Done

**Pull request merged**